README.md

```diff
@@ -665,13 +665,26 @@ Additionally, Pantheon Terminus can be added:
 }
 ```
 
+### Tasks
+
+Tugboat specific tasks are contained in [`tasks/tugboat.yml`](tasks/tugboat.yml).
+Add the following to your `Taskfile.yml`'s `includes` section to use them:
+```yml
+includes:
+  tugboat: ./vendor/lullabot/drainpipe/tasks/tugboat.yml
+```
+|                          |                                                                             |
+|--------------------------|-----------------------------------------------------------------------------|
+| `task tugboat:online`    | Configures Drush with the Tugboat service URL for the environment          |
+
 It is assumed the following tasks exist:
 - `sync`
 - `build`
 - `update`
+- `online`
 
 The `build`, `sync`, and `update` tasks can be overridden with `sync:tugboat`,
-`build:tugboat`, and `update:tugboat` tasks if required (you will need to re-run
+`build:tugboat`,`update:tugboat`, and `tugboat:online` tasks if required (you will need to re-run
```

Comment:
I think that we have to name our things in one way:
`tugboat:something`
or 
`something:tugboat`

We have both ways and that is not consistent.

Reply:
sync, build, and update are pre-existing Task commands and the `:tugboat` that follows it are specific steps for Tugboat. 

The `online` step is very Tugboat specific. When I tried it the other way around I got errors because Taskfile prepends the taskfile name to the nested commands. Thus `tugboat:online`. This got rid of the error.

But yes, I completely agree it doesn't sound consistent and I haven't thought of a better way to address this without a big refactor. Open to ideas and suggestions.

Reply:
A common pattern is to do `<service>:<task>`. Then, all `<service>` tasks are grouped together in help output. This applies both to Task and Symfony / Drush CLI commands!